frontend/packages/console-shared/src/components/modals/ConsolePluginModal.tsx

```diff
@@ -40,19 +39,13 @@ export const ConsolePluginModal = withHandlePromise((props: ConsolePluginModalPr
   return (
     <form onSubmit={submit} name="form" className="modal-content">
       <ModalTitle>
-        {csvPluginsCount > 1
-          ? t('console-shared~Console plugin enablement - {{plugin}}', { plugin: pluginName })
-          : t('console-shared~Console plugin enablement')}
+        {t('console-shared~Console plugin enablement - {{plugin}}', { plugin: pluginName })}
       </ModalTitle>
       <ModalBody>
         <p>
-          {csvPluginsCount
```

Comment:
I think we want to keep the ternary here so the verbiage is more precise since the modal can be launched from either the CSV details page or the Dynamic Plugins list page.

Reply:
got it!